chatwoot/chatwoot
# feat: Ability to specify a subject line for outbound emails

## Description

This feature allows the user to set the subject for outbound emails. Subject field will be displayed only for Email inbox.

Fixes #2306

## Type of change


## How Has This Been Tested?

1. Create new Email inbox.
2. Go to Contacts page. Select one contact. 
3. Click on new message. 
4. Choose Email inbox from drop down.
5. Enter subject, message and click send message.
6. Verify the outbound mail subject using LETTER_OPENER env variable.


![Screenshot 2021-10-08 at 12 48 38 PM](https://user-images.githubusercontent.com/5848740/136514703-45138021-4de2-4c86-84ee-19ed17f4405d.png)
![Screenshot 2021-10-08 at 12 49 50 PM](https://user-images.githubusercontent.com/5848740/136514722-1a8b6070-da59-4f98-86f5-20f4a6584419.png)

## Checklist:
